certified-connectors/Impower/apiDefinition.swagger.json

```diff
@@ -1,3188 +1,3682 @@
-                        "in": "query",
-                        "description": "Sorting direction. Possible values: ASC, DESC",
-                        "x-ms-summary": "Sorting direction. Possible values: ASC, DESC",
-                        "required": false,
-                        "type": "string",
-                        "enum": [
-                            "ASC",
-                            "DESC"
-                        ]
-                    },
-                    {
-                        "name": "page",
-                        "in": "query",
-                        "description": "Page number to be returned",
-                        "x-ms-summary": "Page number to be returned",
-                        "required": false,
-                        "type": "integer",
-                        "format": "int32"
-                    },
-                    {
-                        "name": "size",
-                        "in": "query",
-                        "description": "Number of items to be returned in single page",
-                        "x-ms-summary": "Number of items to be returned in single page",
-                        "required": false,
-                        "type": "integer",
-                        "format": "int32"
-                    },
-                    {
-                        "name": "sort",
-                        "in": "query",
-                        "description": "Sorting parameter",
-                        "x-ms-summary": "Sorting parameter",
-                        "required": false,
-                        "type": "string"
-                    }
-                ],
-                "responses": {
-                    "200": {
-                        "description": "OK",
-                        "x-ms-summary": "OK",
-                        "schema": {
-                            "$ref": "#/definitions/PageOfContactDto"
-                        }
-                    }
-                },
-                "deprecated": false
+        "description": "The body of the webhook"
+      },
+      "post": {
+        "tags": [
+          "Connection"
+        ],
+        "summary": "Create an application connection",
+        "operationId": "createConnectionUsingPOST",
+        "x-ms-trigger": "single",
+        "x-ms-visibility": "important",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "connectionCreateDto",
+            "description": "Connection data.\nid | name | description\n---|---|---\n4 | Powerautomate | Microsoft connected applications\n8 | REST App | Use 8 for generic REST applications",
+            "x-ms-summary": "Connection data.\nid | name | description\n---|---|---\n4 | Powerautomate | Microsoft connected applications\n8 | REST App | Use 8 for generic REST applications",
```

Comment:
Can you please reduce the characters? the swagger validation triggered below error. 

##[error]The 'x-ms-summary' value goes over the character limit '80' ValuePath: /paths/~1v2~1connections/post/parameters/0/x-ms-summary

Reply:
reduced description length.